invenio_app_ils/ui/src/invenio_app_ils/common/api/series/series.js

```diff
@@ -139,9 +151,11 @@ const count = query => {
 
 export const series = {
   url: apiURL,
+  create: create,
   get: get,
   delete: del,
   patch: patch,
+  put: put,
```

Comment:
I would expose the `HTTP verb`: it is better to expose APIs that are abstracting the internal implementation/action: `put` -> `update/edit`?

Probably you can now remove `patch` both here and in the config.py `RECORDS_REST_ENDPOINTS`